lib/widgets/screens/profile/about_my_business.dart

```diff
@@ -1,11 +1,16 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
-import 'package:url_launcher/url_launcher.dart';
 import 'package:mm_flutter_app/constants/app_constants.dart';
 import 'package:mm_flutter_app/widgets/molecules/profile_chip_padded.dart';
+import 'package:url_launcher/url_launcher.dart';
 
 class AboutMyBusiness extends StatelessWidget {
-  const AboutMyBusiness({super.key});
+  final CompanyInput companyInput;
```

Comment:
Is this type from the GraphQL types? If so, it should be a `Company` and not a `CompanyInput`. 

Generally, we only want to use the `Input` types when sending a mutation. This screen seems like it's meant to display the info about the company, not a form which we can use to update the company info.